update.sh

```diff
@@ -46,16 +46,26 @@ echo "version=$Version" >>"$GITHUB_OUTPUT"
 CleanVersion=$(echo "$PreviousVersion" | tr -cd '[:alnum:]._-')
 echo "previous_version=$CleanVersion" >>"$GITHUB_OUTPUT"
 
-PreviousParts=("${CleanVersion//./ }")
+IFS='.' read -ra PreviousParts <<<"$CleanVersion"
```

Comment:
Don't hold up for this comment, but do you need the `-r`? Are we expecting backslashes here?

Reply:
Good question. No, we're not expecting backslashes. Copilot wrote this for me 😅